Library/Models/SurveyCounts.cs

```diff
@@ -32,6 +29,16 @@ public class SurveyCounts
         public int? RejectedCount { get; set; }
 
         /// <summary>
+        /// Number of currently active non-test interviews
```

Comment:
Replace "non-tests" with "live" so description is more direct and consistent with property name